habitat/tasks/rearrange/multi_task/rearrange_pddl.py

```diff
@@ -37,7 +37,7 @@
     from habitat.datasets.rearrange.rearrange_dataset import RearrangeDatasetV0
 
 
-def parse_func(x: str) -> Tuple[str, List[str]]:
+def parse_func(x: str) -> Tuple[str, str]:
```

Comment:
I also fixed this in this PR: https://github.com/facebookresearch/habitat-lab/pull/883